src/components/PickRepoList.tsx

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from '@tanstack/react-query';
+import { getRepos } from '../services/apiService';
+import languageColors from '../json/languageColors.json';
+import { ILanguageColors } from '../models';
+import { Link, useParams } from '@tanstack/react-router';
+
+const PickRepoList = () => {
+  const languageColorsData: ILanguageColors = languageColors;
+  const { username } = useParams({ strict: false });
+  const RepoQuery = useQuery({
+    queryKey: [`getRepos`],
```

Comment:
Just something small to consider: include the username in the queryKey array so that it's unique to that query's data. That way you won't have multiple user's data related to the single getRepos queryKey

Reply:
Oh, thats an interesting concept, let me think on that. Thank you